app/views/layouts/mailer.html.erb

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
Do we need this file

Reply:
for some reason **RSpec** won't run without mailer templates

<img width="1276" alt="Screenshot 2020-03-29 at 19 09 09" src="https://user-images.githubusercontent.com/15877982/77854105-aea24980-71d7-11ea-8815-c4916eb1ea7f.png">

Reply:
@nronas

Reply:
yeap its because of this spec/spec_helper.rb lets leave them for now and we can clean them up later

Reply:
alright